packages/ui-spinner/src/Spinner/index.tsx

```diff
@@ -55,6 +56,7 @@ class Spinner extends Component<SpinnerProps> {
   }
 
   ref: Element | null = null
+  private readonly titleId?: string | undefined
```

Comment:
The second part is not needed because the `?` already makes it optional :)

```suggestion
  private readonly titleId?: string
```